### DISCUSSION TITLE
Prisma with Mongodb -----> trying to create model

### DISCUSSION THREAD

**Discussion by DhariniParker** [prisma/prisma#19602] [Q&A]
### Bug description

Error: MongoDB error
An error occurred during DNS resolution: proto error: Label contains invalid characters: Err(Errors { invalid_mapping, disallowed_by_std3_ascii_rules })

### Expected behavior

It should create collection User with given schema on Mongodb

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```
```prisma
model User {
  id  String @id @default(auto()) @Map("_id") @db.ObjectId
  name String?
  image String?
  email String @unique
}
```
### Environment & setup

- OS: Windows
- Database:  MongoDB
- Node.js version: 18

### Prisma Version

```
4.15.0
```

**Comment by ludralph** [comment]
Hi @DhariniParker 👋 

Can you change `@Map` to `@map` in your schema?

**Reply by nurul3101** [reply]
@DhariniParker Please let us know in case you still need any assistance.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have the a similar problem 🙏
